src/js/package.json

```diff
@@ -28,7 +28,6 @@
     "fast-crc32c": "2.0.0",
     "js-yaml": "^3.14.0",
     "mocha": "7.1.1",
-    "prettier": "2.0.5",
```

Comment:
don't you need the 2.3.x version here?

Reply:
pending you freeze the dependency, LGTM.

Reply:
when we run `npx prettier`, it vaidates if that library is installed, if not, it installs before use, but @graphcareful we should specify `"prettier": "2.3.0"` in order to run always we same version, and avoid conflicts between version

Reply:
correct. we should have static versions for the formatter that match CI 

cc: @ivotron

Reply:
agree. So instead of deleting it from here, we should pin this and the one from GHA to the same version. The GHA workflow installs whatever is latest, so we should pin the same version (specifically here:  https://github.com/vectorizedio/redpanda/blob/dev/.github/workflows/lint.yml#L62). 

btw, should we also do this for `src/v/dashboard/packages.json`?

Reply:
Nice feedback!